pinot-common/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -73,7 +73,74 @@ SqlInsertFromFile SqlInsertFromFile() :
     }
 }
 
-/* define the rest of the sql into SqlStmtList
+
+/**
```

Comment:
Can we enforce this to be only at the end or beginning of the query ?

Reply:
yeah it is already enforces by this syntatic requirement
```
StatementList:
    Statement [; Statement]*

Statement:
    SELECT_STATEMENT | OPTION_STATEMENT | INSERT_STATEMENT
```

because we only allow 1 executable statement inside one SQL string. the OPTION statement can only be at the front or back. do you think this syntatic structure is clear enough to be part of the parser.jj comment/doc? (I will add it to the section if so)